spec/controllers/plug/site_notices_controller_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'rails_helper'
+
+module Plug
+  RSpec.describe SiteNoticesController, type: :controller do
+    routes { Plug::Engine.routes }
+
+    # This should return the minimal set of attributes required to create a valid
+    # site_notice. As you add validations to Site Notice, be sure to
+    # adjust the attributes here as well.
+    let(:valid_attributes) { attributes_for(:site_notice) }
+    let(:invalid_attributes) { attributes_for(:site_notice, :invalid) }
+    let(:site_notice) { create(:site_notice) }
+
+    describe 'GET #index' do
+      it 'returns a success response' do
```

Comment:
It might be a good idea to test the assigns and response type.

Reply:
👍